numba_extensions/pyproject.toml

```
@@ -1,5 +1,50 @@
 # SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "numba_extensions"
+version = "0.0.1"
+description = "Numba extensions for exotic data types in Numba"
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+dependencies= [
+    "numba>=0.59.0",
+    "ast_canopy>=0.0.1",
+    "numbast>=0.0.1"
+]
+
+[project.urls]
+homepage = "https://github.com/NVIDIA/numbast"
+repository = "https://github.com/NVIDIA/numbast"
+
+[project.optional-dependencies]
+dev = ["ruff"]
+
+[tool.setuptools]
+license-files = ["LICENSE"]
```

We should use the root level LICENSE file I think?
```suggestion
license-files = ["../LICENSE"]
```